distributed/ddp-tutorial-series/slurm/setup_pcluster_slurm.md

```diff
@@ -0,0 +1,51 @@
+# Setup AWS cluster with pcluster
+
+## 1. Sign in to an AWS instance
+
+## 2. Install pcluster
+```
+pip3 install awscli -U --user
+pip3 install "aws-parallelcluster" --upgrade --user
+```
+
+## 3. Create a cluster config file
+```
+pcluster configure --config config.yaml
```

Comment:
@hudeven @shaojing @d4l3k when this gets merged it'll be the first recipe we have in examples. Pointing it out because it'd be interesting to do more of this with some longer term goals in mind

Reply:
I'm reusing [torchrecipes repo](https://github.com/facebookresearch/recipes) for Paved Path recipes and plan to make it flat structure as pytorch/examples. The main difference between the 2 repos would be recipes in torchrecipes are more end-to-end and flexible on dependencies, while examples are basic and mainly depend on pytorch. How about moving these tutorials to torchrecipes eventually?